src/Collection/CollectionInterface.php

```diff
@@ -95,6 +95,14 @@ public function filter(?callable $callback = null): CollectionInterface;
      */
     public function reject(callable $callback): CollectionInterface;
 
+    /**
+     * Returns unique elements
+     *
```

Comment:
It would be good to document that uniqueness is based on the string representation of the value and that both `__toString()` and `print_r()` will be used to generate a string to identify values with.